scene/3d/physics_joint_3d.cpp

```diff
@@ -62,18 +62,17 @@ void Joint3D::_body_exit_tree(const ObjectID &p_body_id) {
 }
 
 void Joint3D::_update_joint(bool p_only_free) {
-	if (joint.is_valid()) {
-		if (ba.is_valid() && bb.is_valid()) {
-			PhysicsServer3D::get_singleton()->body_remove_collision_exception(ba, bb);
-		}
-
-		PhysicsServer3D::get_singleton()->free(joint);
-		joint = RID();
-		ba = RID();
-		bb = RID();
+	if (ba.is_valid() && bb.is_valid()) {
+		PhysicsServer3D::get_singleton()->body_remove_collision_exception(ba, bb);
 	}
 
+	ba = RID();
+	bb = RID();
+
+	configured = false;
+
```

Comment:
This code needs condition `if (is_configured())` to replace the previous `if (joint.is_valid())`.

Reply:
technically, ba and bb are assigned only if configured so it should not be a problem.